Documentation/user-guides/getting-started.md

```diff
@@ -276,6 +270,9 @@ spec:
 
 This enables the frontend team to create new ServiceMonitors and Services which allow Prometheus to be dynamically reconfigured.
 
+Note: `serviceMonitorSelector: {}` selects all service monitors.  If you omit this key altogether, it won't auto-detect any 
```

Comment:
Here the other methods should be mentioned as well (as well as their effect):

* Omitting the `serviceMonitorSelector` or setting it to `null` enabled one to use a [custom configuration](https://github.com/coreos/prometheus-operator/blob/master/Documentation/custom-configuration.md).
* `serviceMonitorSelector: {}` selects all ServiceMonitors in the same namespace as the Prometheus object.
* Specifying a selector selects exactly those ServiceMonitors (in the same namespace as the Prometheus object) with the matching selectors. Find the documentation for selectors
 [here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#labelselector-v1-meta) and usage guide [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements).

Something along those lines.

Also all the text goes on a single line, as a paragraph, otherwise rendering these on a website gets weird.